services/worker/src/worker/loop.py

```diff
@@ -124,12 +123,7 @@ def process_next_job(self) -> bool:
                 logging.debug(e)
                 return False
 
-        buckets = (
-            DESCRIPTIVE_STATISTICS_PROMETHEUS_HISTOGRAM_BUCKETS
-            if job_info["type"] == "split-descriptive-statistics"
-            else None
-        )
-        with StepProfiler("loop", "run_job", job_info["type"], buckets=buckets):
+        with LongStepProfiler("loop", "run_job", job_info["type"]):
```

Comment:
so we do this for all workers now, right?

Reply:
yes